Documentation/guides.xml

```diff
@@ -9,6 +9,8 @@
         class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
         edit-on-github="TYPO3-Documentation/render-guides"
         edit-on-github-branch="main"
+        official-manual-interlink-shortcode="t3docsRenderGuides"
```

Comment:
Hmm, not convinced that this is the way: When I use this, for example, in my EXT:schema (like "ext_schema" as shortcode). Then the user who uses this reference in their extension's docs has to configure it manually.

This may be fine, but we have to hint about this in the modal.

Reply:
I will take care of this in a follow up. I first want to investigate on automatic linking to extension manuals. If that proves not to be possible we can still add more info here and reintroduce the flag weather a manual is predefined